tests/test_interface.rb

```diff
@@ -780,7 +785,8 @@ def test_negotiate_auto_ethernet
     rescue Cisco::CliError => e
       skip('Skip test: Interface type does not allow config change') if
         e.message[/requested config change not allowed/]
-      flunk(e.message)
+      flunk(e.message) unless
+        e.message[/cannot turn off auto-negotiate with auto-negotiate speeds/]
```

Comment:
It looks like the message is related to a possible bug. I'll pull out the change for now